core/modules/system/system.install

```diff
@@ -538,6 +538,30 @@ function system_requirements($phase) {
       );
     }
   }
+  
+  if ($phase == 'runtime') {
+    // Inform user about status of display error messages.
+    $error_level = config_get('system.core', 'error_level');
+    $description_start = $t('Backdrop CMS provides the ability to display error messages, which can be useful while a site is in development.');
+    $logging_url = base_path() . 'admin/config/development/logging';
+    if ($error_level != ERROR_REPORTING_HIDE) {
+      $value =  $t('Enabled');
+      $severity =  REQUIREMENT_WARNING;
+      $description = $description_start . ' ' . $t('Do not forget to <a href="@error_level">disable the display of errors</a> when this site is in use.', array('@error_level' => $logging_url));
```

Comment:
The point is to avoid the word "production". Can you think of a better word/phrase than "in use"?  Most normal people won't know what production means.